### PR TITLE
Add activeArea attribute

### LINKED ISSUE #11: Addition ittp:activeArea
The following change needs to be reflected in the EBU-TT-D XSD for version 1.0.1.

### Before
This feature and attribute was not present.
### After
The attribute was added to the tt:tt element.

### PR DESCRIPTION
**Pull request by andreastai**
- This closes #11.

### CODE REVIEW

**Review by nigelmegitt — APPROVED** [review]
Tested (manually) and works. Looks good to me.

**Comment by andreastai** [comment]
Thanks, @nigelmegitt I will merge it after the minimum review period has passed and no other review blocks merging.

**Comment by andreastai** [comment]
As the PR is open for more than 10 working days and there are no blocking reviews, it will be merged (after merge conflicts have been resolved).

**Pull request merged**
